.jenkins

```diff
@@ -312,6 +314,9 @@ pipeline {
                     }
                     environment {
                         OMP_NUM_THREADS = 8
+                        // Nested OpenMP does not work for this configuration,
+                        // so disabling it
+                        OMP_MAX_ACTIVE_LEVELS = 1
```

Comment:
The problem here is that OpenMP reports that we can use nested parallel region but this [region](https://github.com/kokkos/kokkos/blob/develop/core/src/OpenMP/Kokkos_OpenMP_Exec.cpp#L212-L238) is only executed by a single thread. I have tried to disable UVM and RDC but that's not the problem. There is a similar problem with the other configuration but there setting `OMP_MAX_ACTIVE_LEVELS` fixes the problem.

Reply:
Is there any issue tracking this problem?

Reply:
No but the feature is already deprecated and only used by UIntah. I only found out about this test because I am refactoring the code, so that we have an alternative to `partition_master` and we can get ride of it. The new code won't use nested parallelism so we won't have the problem.